src/main/resources/atlassian-plugin.xml

```diff
@@ -149,6 +149,20 @@
         <exporter class="org.jfrog.bamboo.exporter.ArtifactoryTaskExporter"/>
     </taskType>
 
+    <!--Collect issues task-->
+    <taskType key="artifactoryCollectIssuesTask" name="Artifactory Collect Issues"
+              class="org.jfrog.bamboo.task.ArtifactoryCollectIssuesTask">
+        <description>Artifactory Collect Issues Task</description>
```

Comment:
I suggest renaming the task name to:
**Artifactory Build Issues Task**
If we're going for this change, let's also rename all the relevant methods and variables from ...collectIssues... to ...buildIssues...